rotkehlchen/chain/gnosis/modules/cowswap/constants.py

```diff
@@ -0,0 +1 @@
+CPT_COWSWAP = 'cowswap'
```

Comment:
This should not be gnosis specific. Move this to the `evm/decoding/coswap/constants.py` file

Reply:
Is there two. I copy-pasted too much